app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+
+  def new
+    redirect_to home_path if current_user
+  end
+  
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
Since Rails 4, use `.find_by` instead of `.where(args).first` for single record search. its more effective.